packages/react-vapor/src/components/select/tests/SingleSelectConnected.spec.tsx

```diff
@@ -254,5 +255,25 @@ describe('Select', () => {
                 expect(getIsOpen()).toBe(false, 3);
             });
         });
+
+        describe('footer props', () => {
+            it('should render the footer', () => {
+                const footer = <span> 👢 </span>;
+                const shallowedSingleSelect = shallowWithStore(
+                    <SingleSelectConnected id={id} items={[{value: 'a', selected: false}]} footer={footer} />,
+                    store
+                );
+                expect(
+                    shallowedSingleSelect
+                        .dive()
+                        .dive()
+                        .dive()
+                        .dive()
+                        .childAt(0)
+                        .children()
+                        .prop('footer')
+                ).toEqual(footer);

```

Comment:
Is there a better way to do this? somehow could get anything with .find(selector) since it's shallow mounted.